sass/components/_global.scss

```diff
@@ -401,7 +397,7 @@ td, th{
   display: table-cell;
   text-align: left;
   vertical-align: middle;
-  border-radius: 2px;
+  border-radius: 0;
```

Comment:
I think this line is also safe to delete... what do you think guys? @materializecss/members

Reply:
Yes, agree 👍

Reply:
I would not optimize it more, to avoid the risk of breaking existing applications because of how the styles are used. This is a nice and small pull request as it is now. Let's keep it that way.